InteractiveHtmlBom/errors.py

```diff
@@ -0,0 +1,12 @@
+from sys import exit
+
+ERROR_PARSE = 3
+ERROR_FILE_NOT_FOUND = 4
+ERROR_NO_DISPLAY = 5
+
+class ParsingException(Exception):
+    pass
+
+def exit_error(logger, code, err):
```

Comment:
If you are passing exception as third parameter it should be named `ex` or `exception`.
I'm not convinced logging whole exception is good, just message will be the same information and is more readable to non-programmers.

Also formatting nit: two empty lines between top level blocks in a module.

Reply:
Now I'm passing a string. But I don't see any difference between `print(e)` and `print(str(e))` both print the first argument passed during the object creation in the raise statement. I get a traceback only if I don't catch the exception.

Reply:
logger methods are not the same as print(). Print internally just calls `__str__` which is what `str()` does as well so you won't see a difference but logger prints stack trace (at least it should on fine enough logging levels).

Reply:
This is *error* level and can't be changed. I get the __str__, at least on Python 3, I can't test with Python 2 because my pcbnew is for Python 3. Which level prints a traceback?